eoscostarica/guide.eoscostarica.io
# Feat(frontend): Reat.JS Tutorial

### GH Issue

Resolves #270  

### Steps to test
1. Go to Tutorials/React JS
2. Check the content is right

#### CheckList
- [ ] The content is equal to https://guias.eoscostarica.io/docs/tutoriales/tutorial-react
